packages/management/framework/bundle/config/pacakgeinstalls/tkr-service-service-account.yaml

```diff
@@ -0,0 +1,156 @@
+#@ load("@ytt:data", "data")
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tkr-service-package-sa
+  namespace: #@ data.values.namespaceForPackageInstallation
+  annotations:
+    kapp.k14s.io/change-group: "tkr-service-packageinstall/serviceaccount-0"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: tkr-service-package-cluster-role
+  annotations:
+    kapp.k14s.io/change-group: "tkr-service-packageinstall/serviceaccount-0"
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+      - serviceaccounts
+      - services
+    verbs:
+      - create
+      - update
+      - get
+      - list
+      - delete
+      - watch
+  - apiGroups:
+      - apps
+    resources:
+      - deployments
+    verbs:
+      - create
+      - update
+      - get
+      - list
+      - delete
+  - apiGroups:
```

Comment:
i think this rule is not needed as tkr-service package doesn't have any CRD resources